sidebery.json

```diff
@@ -2,19 +2,61 @@
   "settings": {
     "nativeScrollbars": true,
     "nativeScrollbarsThin": true,
-    "ctxMenuNative": true,
-    "navBarLayout": "hidden",
+    "updateSidebarTitle": true,
+    "ctxMenuNative": false,
+    "navBarLayout": "horizontal",
+    "navBarInline": true,
     "subPanelRecentlyClosedBar": false,
     "subPanelBookmarks": false,
     "subPanelHistory": false,
-    "showNewTabBtns": true,
+    "searchBarMode": "dynamic",
     "tabRmBtn": "always",
+    "tabsRmUndoNote": false,
+    "showNewTabBtns": true,
     "newTabBarPosition": "after_tabs",
     "moveNewTabParent": "default",
     "pinnedTabsPosition": "panel",
     "pinnedTabsList": true,
+    "bookmarksRmUndoNote": false,
     "theme": "proton",
     "colorScheme": "ff"
   },
-  "ver": "5.1.1"
-}
+  "sidebar": {
```

Comment:
What was the reason for adding these panels? As far as I know most of these are there by default?

Reply:
I have removed settings that may be specific to the way I use Sidebery